WordPress/src/main/java/org/wordpress/android/models/SiteSettingsModel.java

```diff
@@ -158,8 +167,9 @@ public boolean equals(Object other) {
                 commentsRequireUserAccount == otherModel.commentsRequireUserAccount &&
                 commentAutoApprovalKnownUsers == otherModel.commentAutoApprovalKnownUsers &&
                 maxLinks == otherModel.maxLinks &&
-                holdForModeration != null && holdForModeration.equals(otherModel.holdForModeration) &&
-                blacklist != null && blacklist.equals(otherModel.blacklist);
+                holdForModeration != null && otherModel.holdForModeration != null
```

Comment:
I don't think `otherModel.holdForModeration != null` is necessary here. The other 2 cases will cover this case. If the `holdForModeration ` is not `null` and equal to `otherModel.holdForModeration`, then it means `otherModel.holdForModeration` is not `null` either.

We could keep it for verbosity though, so the devs reading the code doesn't need to think about it. It's up to you!